liquidpromptrc-dist

```diff
@@ -170,10 +170,16 @@ LP_ENABLE_SCLS=1
 # Show current Kubernetes kubectl context
 LP_ENABLE_KUBECONTEXT=0
 
-# Delimiter to shorten kubectl context.
+# Delimiter to shorten kubectl context by removing a suffix.
 # E.g. when your context names are dev-cluster and test-cluster, set to "-"
 # in order to output "dev" and "test" in prompt.
-LP_DELIMITER_KUBECONTEXT=
+LP_DELIMITER_KUBECONTEXT_SUFFIX=
+
+# Delimiter to shorten kubectl context by removing a prefix.
+# E.g. when your context names are like
+# arn:aws:eks:$REGION:$ACCOUNT_ID:cluster/$CLUSTER_NAME, set to "-"
```

Comment:
This example doesn't work, shouldn't the example value be `/`?

Reply:
oops, yes